haystack_experimental/components/builders/chat_prompt_builder.py

```diff
@@ -1,276 +0,0 @@
-# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
this was only implemented to support the new `ChatMessage` (now integrated in Haystack)